src/templates/prototypes/_forms.pug

```diff
@@ -68,15 +68,16 @@ block content
 
       ol.list--pipe__deep
         li
-          figure.svg--icon(data-file-upload='icon')
-              include ../../icons/file.svg
-          .list--details
-            h3 Attach your petition template
+          .card
```

Comment:
Applying the card sizing options don't work here. Do we also still need svg--icon after applying card amends?